karma.py

```diff
@@ -47,11 +47,22 @@ def karma(self, event):
     @cmd
     def top_karma(self, event):
         """Get 5 people with most karma points."""
+        args = event.args
```

Comment:
@pepol @Adman @JakubNvk  I am wondering whether it would not make sense to make this an optional second parameter to for any `@cmd`. There are quite a lot of these lines in the codebase and I do not think there is a good reason for that.